lib/internal/test_runner/reporter/lcov.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const Transform = require('internal/streams/transform');
+
+// This reporter is based on the LCOV format, as described here:
+// https://ltp.sourceforge.net/coverage/lcov/geninfo.1.php
+// Excerpts from this documentation are included in the comments that make up
+// the _transform function below.
+class LcovReporter extends Transform {
+  constructor(options) {
+    super({ ...options, writableObjectMode: true, __proto__: null });
+  }
+
+  _transform(event, _encoding, callback) {
```

Comment:
why not:
```suggestion
  #transform(event, _encoding, callback) {
```

Reply:
According to the docs, [a Transform stream class must implement a `_transform` method](https://nodejs.org/api/stream.html#transform_transformchunk-encoding-callback). I don't believe you can replace that with a private method, though I'm happy to be proven wrong.

Reply:
Yeah this is a "protected", a relic from the past :D

Reply:
So you can use a private method? Is that something we can update in the docs that I linked to?

Reply:
You can't use a private method, what you did is correct and is the Transform contract, if you use #transform it wouldn't work (by design, due to how private fields work in ECMAScript

Reply:
Ok, cool. Thought I was missing out on something there!